### DISCUSSION TITLE
SKIOVOX QUESTION

### DISCUSSION THREAD

**Discussion by CGRmags** [3kh0/ext-remover#905] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Can someone walk me though the 120 Skiovox way and how to do it? Because nothing happens when I do the chrome vox or the wifi thing. If this isnt possible to fix then that ok but It would be nice to know. Thanks!

**Comment by synapt0** [comment, marked as answer]
Check my reply to your comment on my discussion :)
